### ISSUE TITLE
License inconsistencies

### ISSUE THREAD

**Issue by Octogonapus** [Decave/JCoz#29]
#11 states that this work will use the GPL license (and indeed, a copy of the GPL is in the root directory). However, most files have this text as part of the license header:
```
 * Unless otherwise noted, all of the material in this file is Copyright (c) 2016
 * by David C Vernet and Matthew J Perron. All rights reserved. No part of this file
 * may be reproduced, published, distributed, displayed, performed, copied,
 * stored, modified, transmitted or otherwise used or viewed by anyone other
 * than the authors (David C Vernet and Matthew J Perron),
 * for either public or private use.
 *
 * No part of this file may be modified, changed, exploited, or in any way
 * used for derivative works or offered for sale without the express
 * written permission of the authors.
```

If this work really is GPL, these license headers should be adjusted to conform to the GPL. I am interested in continuing to develop JCoz, but I am hesitant to contribute until these license headers are adjusted.

**Comment by Decave** [comment]
I am currently in Japan until Friday, but have no problem adjusting these header files to conform to GPL. Please feel free to send a PR for that, or I’ll take care of it myself when I’m back in the US.

**Comment by Octogonapus** [comment]
Closed in #34.
